app/src/main/AndroidManifest.xml

```diff
@@ -13,16 +13,26 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
+
         <activity android:name=".ui.splash.SplashScreenActivity"
-            android:theme="@style/SplashScreenTheme">
+            android:theme="@style/SplashScreenTheme"
+            android:noHistory="true">
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
-        <activity android:name=".ui.registration.RegistrationActivity"/>
-        <activity android:name=".ui.registration.RegistrationConfirmationActivity"/>
-        <activity android:name=".ui.main.DashboardActivity"/>
+
+        <activity android:launchMode="singleTask"
+            android:name=".ui.registration.RegistrationActivity"/>
```

Comment:
W międzyczasie weszło OnboardingActivity, które jest pierwszym z trzech activity rejestracji. Czy mógłbyś domerge'ować zmiany i to dostosować?

Reply:
Zrobiony rebase, dodana obsługa onboardingu